src/VSCodeExtension/syntaxes/qsharp.tmLanguage.json

```diff
@@ -2,7 +2,6 @@
   "copyright": "Copyright ⓒ 2017-2019 Microsoft Corporation. All Rights Reserved.",
```

Comment:
```suggestion
```
Hmm, apparently the script Linguist uses to add grammars also complains about an "unexpected" field, `copyright`. The [starter file](https://github.com/microsoft/vscode-generator-code/blob/master/generators/app/templates/ext-language/syntaxes/new.tmLanguage.json) doesn't include that field, and if I remove it here, together with the version number (see: [my fork](https://github.com/JakuJ/qsharp-compiler/blob/3b1d2015ab7b45e223d0acb2ad6a9946fe0d4cce/src/VSCodeExtension/syntaxes/qsharp.tmLanguage.json#L2)), only then does it get detected properly.